Clarify requirements (#8)
freight-hub/freighthub-frontend-challenge: Criteria.md

## Must Have

- The application must start
- The application must be a single-page application (SPA)
- The application must be divided into components
- The file structure must be consistent and easy to follow
- The list and details page must cover all [use cases](README.md#use-cases) with minor issues

> - The application must include tests which validate logic or component behavior (The built-in test from Create React App does not count, and neither do snapshot tests)

Reviewer: ```suggestion
- The application must include tests which validate logic or component behavior (Tests that does not document/validate the logic and behavior, eg snapshot testing, are no valid)
```

Author: ```suggestion
- The application must include meaningful tests (Tests that do not validate logic or component behavior, e.g. snapshot tests, are not counted)
```
